### PR TITLE
4 create list contact UI

### PR DESCRIPTION
**Pull request by abhirampai**
fixes issue #4 ,

* Created Scrollable div to list contacts.
* Reused the menubar. 
* Added delete contact functionality.
* Added avatar and other typography components as required.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      createdAt: "Oct 14th, 2021"
+    },
+    {
+      id: 3,
+      name: "Jacob Johns",
+      role: "Owner",
+      email: "jacobjohns@example.com",
+      createdAt: "Oct 14th, 2021"
+    }
+  ]);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:47]: Please move this to a constants file.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+import { Header, Scrollable, Container } from "neetoui/v2/layouts";
+
+import EmptyState from "components/Common/EmptyState";
+import Menubar from "components/Common/Menubar";
+
+import ContactRow from "./ContactRow";
+
+const Contacts = () => {
+  const [loading, setLoading] = useState(false);
+  // const [showNewContactPane, setShowNewContactPane] = useState(false);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:22]: Please remove this if not needed.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                      <tbody>
+                        {contacts.map((contact, index) => (
+                          <ContactRow
+                            key={index}
+                            contact={contact}
+                            deleteContact={deleteContact}
+                          />
+                        ))}
+                      </tbody>
+                    </table>
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:155]: We can move this as a separate component for listing contacts.

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+                        strokeWidth="1.5"
+                        d="M3 11.25L15 11.25"
+                      ></path>
+                      <path
+                        stroke="#68737D"
+                        strokeWidth="1.5"
+                        d="M3 15.25L11 15.25"
+                      ></path>
+                    </svg>
+                  );
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:101]: Commenting on this. We will look into it.

**Comment by abhirampai** [comment]
> @abhirampai Please refrain from adding branch numbers in PR titles.

yea sure will keep that in mind 👍

**Pull request merged**